### PR TITLE
Opamroot redirect

### PR DESCRIPTION
**Pull request by dra27**

### CODE REVIEW

File: src/client/opamClient.ml [hunk 1]
```diff
@@ -1645,6 +1645,111 @@ let reinit ?(init_config=OpamInitDefaults.init_config()) ~interactive
...
+  root_dir
+
+let get_redirected_root () =
+  let {contents = {OpamStateConfig.original_root_dir = root; root_from; _}} =
+    OpamStateConfig.r
+  in
+  let r = OpamConsole.colorise `bold (OpamFilename.Dir.to_string root) in
+  let collision =
+    (* UTF-8 <U+0020, U+1F4A5> (Collision emoji) *)
+    if OpamConsole.color () then " \xF0\x9F\x92\xA5" else ""
```
Comment (rjbou) [inline src/client/opamClient.ml]: This should be defined in OpamConsole

File: src/client/opamClient.ml [hunk 2]
```diff
@@ -1645,6 +1645,111 @@ let reinit ?(init_config=OpamInitDefaults.init_config()) ~interactive
...
+        r
+    | `Quit ->
+      OpamStd.Sys.exit_because `Aborted
+  in
+  OpamConsole.header_msg "opam root file store";
+  OpamConsole.msg
+    "\n\
+     %s\n\
+     \n\
+     Many parts of the OCaml ecosystem do not presently work correctly\n\
```
Comment (rjbou) [inline src/client/opamClient.ml:1751]: maybe a less ocaml centric message ?

Comment (dra27) [reply src/client/opamClient.ml:1751]: I see where you're coming from, but we do already have "opam and the OCaml ecosystem in general require various Unix tools" in the previous messages?

Comment (rjbou) [reply src/client/opamClient.ml:1751]: true

File: src/client/opamClient.ml (hunk 1 above)
Comment (dra27) [reply src/client/opamClient.ml]: Blimey, yes - see https://github.com/dra27/opam/pull/22/commits/725e63c7594f4804489ec359794f13d0181ce5ff!

Comment (dra27) [reply src/client/opamClient.ml]: (also verified that that means the running `opam init` in conhost correctly doesn't display anything)

**Pull request closed without merging**